MeNext/Layout/ResultList/ListsView.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+
+using Xamarin.Forms;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+
+using MeNext.MusicService;
+using System.Diagnostics;
+
+namespace MeNext
+{
+    /// <summary>
+    /// This is a view which allows us to EITHER (1) display multiple lists of metadata with a title for each grouping
+    /// OR (2) display a single incomplete list whose values are loaded dynamically. One cannot have multiple
+    /// dynamically loaded lists.
+    /// </summary>
+    public class ListsView<T> : ListView where T : IMetadata
+    {
+        //private IResultList<T> resultList;
+        //private BetterObservableCollection<ResultItemWrapper<T>> resultCollection;
+        //ResultItemFactory<T> resultItemFactory;
+
+        private List<ResultsGroup<T>> allGroups;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:MeNext.QueuesView`1"/> class. This constructor is for
+        /// multiple lists of elements, represented by GroupWrappers.
+        /// </summary>
+        /// <param name="groups">The list of GroupWrappers</param>
+        public ListsView(params ResultsGroup<T>[] groups)
+        {
+            this.HasUnevenRows = true;
+
+            this.IsGroupingEnabled = true;
+            this.GroupDisplayBinding = new Binding("Title");  // TODO
```

Comment:
Yeah, I hate bindings and avoided their use as much as possible. That TODO with no helpful information is actually for me to look into using reflection witchcraft to be less fragile.